distribute-with-merkl/types-of-campaign/README.md

```diff
@@ -0,0 +1,101 @@
+
+## 2. Airdrop Campaign
+
+* For a detailed guide on how to set up your airdrop campaign, check out this [page](airdrop-campaign.md).
+
+{% content-ref url="airdrop-campaign.md" %}
+[airdrop-campaign.md](airdrop-campaign.md)
+{% endcontent-ref %}
+
+* When you're ready to start your airdrop campaign, create it using [Merkl's Airdrop Campaign creation tool](https://app.merkl.xyz/create/drop). **Before you begin, do not forget to whitelist the tokens you want to distribute in the airdrop by filling out this** [form](https://tally.so/r/3y2bqx).
+
+## 3. ERC20 Incentivization Campaign (Token, LP Token, and Lending and Borrowing Token Balance)
+
+ERC20 campaigns incorporate various aspects such as token balance, LP token balance (liquidity pool V2), and lending and borrowing protocols. Since all these protocols are based on ERC20 tokens, Merkl can integrate them by default, ensuring that users participating in different activities can earn rewards.&#x20;
+
+However, for V2 AMMs and lending and borrowing protocols, **we strongly recommend you to be fully supported by Merkl**. The advantages of being supported by Merkl include APR and TVL calculations, among other benefits. Additionally, you can create protocol-specific or campaign-specific types of campaigns for more tailored incentives.
+
+The current supported lending and borrowing protocols by Merkl are:
+
+* Silo (protocol-specific campaigns)
+* Radiant (protocol-specific campaigns)
+* Compound
+* Gearbox
+* Ionic
+* Sturdy
+* Metamorpho Lending
+
+Merkl also currently support the following V2 AMMs:
+
+* Uniswap V2
+* Balancer V2
+* Velodrome V2
+* Aerodrome V2
+* Fenix V2
+* Poolside V2
+* Quickswap V2
+
+if you want your AMM V2, or lending and borrowing protocol to be fully integrated and supported by Merkl, or to create protocol or campaign-specific campaigns (as we did for Silo and Radiant), please [contact us on the Merkl Discord by opening a BD ticket](https://discord.com/invite/jnYfrGxDbe) to discuss the integration process. Integration allows APRs and TVL calculations.&#x20;
+
+* For a detailed guide on how to set up your ERC20 campaign, check out this [page](erc20-incentivization-campaign.md).
+* When you're ready to start incentivizing an ERC20 campaign, create your incentivization campaign using [Merkl's ERC20 Campaign creation tool](https://app.merkl.xyz/create/hold). **Before you begin, do not forget to whitelist the tokens you want to incentivize the liquidity pool with, by filling out this** [form](https://tally.so/r/3y2bqx).
+
+## 4. Lending and Borrowing Incentivization Campaign on Silo and Radiant
```

Comment:
not sure it's really useful to have nearly the same text repeated. We could do something generic, no ?